qos/jsonrpc/params.go

```diff
@@ -0,0 +1,42 @@
+package jsonrpc
+
+import (
+	"encoding/json"
+	"errors"
+)
+
+// Params stores the data contained in the `params` field of a JSONRPC request.
+// As of PR #170, it supports:
+// - An array of objects
+// - An array of strings
+// - An array of primitive types
+// See the below link on JSONRPC spec for more details:
+// https://www.jsonrpc.org/specification#parameter_structures
+type Params struct {
+	// Params stores the raw JSON-RPC parameters without validation.
+	// The method-specific request handlers are responsible for validating the parameters based on the JSON-RPC method being called.
```

Comment:
Could you update with a comment on what `rawPayload` is.

An example would be grat.